skynet/utils.py

```diff
@@ -112,6 +110,7 @@ def pipeline_for(
         return custom_pipeline.pipeline_for(model, mode, mem_fraction=mem_fraction, cache_dir=cache_dir)
 
     except ImportError:
+        # TODO, uhh why not warn/error log this?
```

Comment:
Currently most of our models use the generic pipeline, so the common case is that the `ImportError` gets raised, we should log something like "custom pipeline for {model_info.name} not found, loading with generic pipeline"

Reply:
Sounds good, defeinetly need to improve daemon comms to be able to do complex behaviors